ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-postsubmits.yaml

```diff
@@ -10,6 +10,7 @@ postsubmits:
       skip_cloning: true
     labels:
       ci.openshift.io/generator: prowgen
+      job-release: "4.13"
```

Comment:
Did you introduce changes to these files manually or were they autogenerated?

Reply:
Autogenerated. And goes to my comment on the 'releases' stanza earlier.